code/game/machinery/doors/airlock.dm

```diff
@@ -928,7 +928,7 @@ About the new airlock wires panel:
 					src.unlock(TRUE) //force it
 
 /obj/machinery/door/airlock/CanUseTopic(var/mob/user)
-	if(operating < 0) //emagged
+	if(emagged == 1)
```

Comment:
```suggestion
	if(emagged)
```

Emagged is binary right? So can just have this.

Reply:
I checked, and emagged is seen with with numbers like 2 and 0.5, so I assumed I should keep it to 1 and 0. Is that wrong?